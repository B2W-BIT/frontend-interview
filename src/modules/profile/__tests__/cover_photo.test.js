import React from 'react';
import CoverPhoto from '../components/cover_photo'
import renderer from 'react-test-renderer'

describe("Will test the coverphoto component", () => {

    it('renders de cover photo nicely', () => {
        const tree = renderer.create(
            <CoverPhoto photo="https://www.papeldeparede.etc.br/fotos/wp-content/uploads/gatinho_fofinho_na_rede.jpg" />
        ).toJSON();
        expect(tree).toMatchSnapshot()
    })
})
