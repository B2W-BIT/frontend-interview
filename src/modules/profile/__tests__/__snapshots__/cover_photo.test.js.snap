exports[`Will test the coverphoto component renders de cover photo nicely 1`] = `
<div
  className="cover_photo"
  style={
    Object {
      "backgroundImage": "url(undefined)",
    }
  } />
`;
